test/utils.ts

```diff
@@ -12,8 +14,15 @@ export const getConfig = (filename: string): InlineConfig => ({
   configFile: fileURLToNormalizedPath(new URL(`./${filename}`, root))
 })
 
-export const loadFileContent = async (path: string) => {
+export const loadFileContent = async (
+  path: string,
+  encoding: BufferEncoding = 'utf8'
+): Promise<string | ArrayBuffer> => {
   const absolutePath = new URL(path, root)
-  const content = await readFile(absolutePath, 'utf8')
+  const content = await readFile(absolutePath, encoding)
+
+  if (encoding === 'binary') {
+    return toArrayBuffer(content)
+  }
   return content
```

Comment:
```suggestion
  const absolutePath = new URL(path, root)
  if (encoding === 'binary') {
    const content = await readFile(absolutePath)
    return content.buffer
  }

  const content = await readFile(absolutePath, encoding)
  return content
```
I think this will work. TIL `fs.readFile(path, 'binary')` returns a `string`. (and I noticed there was a bug in the previous PR that `encoding: 'binary'` returns a string.)

Reply:
I tried `content.buffer` but was `undefined`

Reply:
done!